acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/models/Publication.java

```diff
@@ -62,6 +67,18 @@ public Instant getNominalDate() {
         return nominalDate;
     }
 
+    public String getNominalDateFormatted() {
```

Comment:
This seems risky. I'd rather see specific assertions around the nominal date stuff, rather than a reimplementation of the logic, which could itself be wrong.

Reply:
Discussed, agreed leaving as is.